service/iot-service-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/service/RegistryManagerTest.java

```diff
@@ -1162,7 +1163,74 @@ public Device getStatistics() throws IOException, IotHubException
         registryManager.getStatisticsAsync();
     }
 
+    @Test
+    public void exportDevices_jobProperties_good_case() throws Exception
+    {
+        String connectionString = "HostName=aaa.bbb.ccc;SharedAccessKeyName=XXX;SharedAccessKey=YYY";
+
+        new NonStrictExpectations()
+        {
+            {
+                IotHubConnectionStringBuilder.createConnectionString(connectionString);
+                result = iotHubConnectionString;
+                iotHubConnectionString.getUrlCreateExportImportJob();
+                result = mockUrl;
+                mockHttpRequest.send();
+                result = mockHttpResponse;
+                mockIotHubExceptionManager.httpResponseVerification((HttpResponse) any);
+                mockHttpResponse.getBody();
+                result = jobPropertiesJson.getBytes();
+            }
+        };
+
+        RegistryManager registryManager = RegistryManager.createFromConnectionString(connectionString);
+        JobProperties exportJobProperties =
+                JobProperties.createForExportJob("blob1", true, StorageAuthenticationType.IDENTITY);
+        JobProperties jobProperties = registryManager.exportDevices(exportJobProperties);
```

Comment:
So where are you setting the env for this to work?

Reply:
For .NET, I'm setting it locally when I test.

For our pipeline, we can't set the env var because this api-version will not be rolled to west us, so the test would fail. We'll have to wait for the feature to make it to west us, and enable our hubs with identity (still a TODO for me, so I created this PBI: https://msazure.visualstudio.com/One/_boards/board/t/IoT-Developers%20and%20Devices-Managed/Backlog%20items/?workitem=6454493).

For now, we're relying on local testing to validate the new functionality.